### DISCUSSION TITLE
Custom LXMF addres for 'Local Peer Info' / Operator

### DISCUSSION THREAD

**Discussion by PoincareMesh** [markqvist/Reticulum#427] [Requests for New Features]
I am running a Nomadnet node in daemon mode on an 'always on' server but I read my messages on my desktop. If someone wants to contact me about the node, they may use the LXMF address associated with the node, but those messages may never be read. It would be nice if we could configure an alternate (or forward?) LXMF operator address on the node so that messages will go to a place where they will get noticed.

**Comment by markqvist** [comment]
Currently, nomadnet calculates the matching operator LXMF address from the node address and public identity keys. There's not any provision for including a custom operator address in the announce, so the best option right now is simply to make an LXMF link on one of the pages of your node - that way people can at least just click on that to get in contact.

A possible future feature would be to allow forwarding messages to the operator address to another address. It's not something I can personally prioritize right now, but if anyone else wants to implement it, that'd be a good candidate for a PR :)

**Reply by faragher** [reply]
My mailbox utility has similar functionality; we could probably repurpose it from a mailbox to an automated proxy once it's released. Since it's compatible with the way NomadNet stores files, it should be reasonably straightforward, especially if we add a hook in NN itself.
